### PR TITLE
update ideaCard schema

### PR DESCRIPTION
**Pull request by alicefate**
add:
extraInfo 
status
involveLevel
ideaOwner

delete:
hourCommitmentMin*	
hourCommitmentMax*
difficultyLevel*

### CODE REVIEW

File: api/idea-card/models/idea-card.settings.json [hunk 1]
```diff
@@ -75,10 +58,39 @@
...
-      "type": "string"
+      "type": "string",
+      "required": true
     },
     "experience": {
-      "type": "string"
+      "type": "string",
+      "required": true
+    },
+    "extraInfo": {
```
Comment (chungthuang) [inline api/idea-card/models/idea-card.settings.json:68]: How will this be used?

File: api/idea-card/models/idea-card.settings.json [hunk 2]
```diff
@@ -75,10 +58,39 @@
...
+        "applying",
+        "approved",
+        "archived",
+        "workshopping"
+      ]
+    },
+    "involveLevel": {
+      "type": "enumeration",
+      "enum": [
+        "one",
```
Comment (chungthuang) [inline api/idea-card/models/idea-card.settings.json:84]: Would it be easier to understand if we make this more descriptive? Like `minimum`, `medium`, `highly` etc?

File: api/idea-card/models/idea-card.settings.json (hunk 1 above)
Comment (alicefate) [reply api/idea-card/models/idea-card.settings.json:68]: In submission form,  there are four required questions, ideaName, description, experience and features. Frontend will check user input before it send to servers.

File: api/idea-card/models/idea-card.settings.json (hunk 2 above)
Comment (alicefate) [reply api/idea-card/models/idea-card.settings.json]: good idea. changed them to "none, minimum, medium, highly"

File: api/idea-card/models/idea-card.settings.json (hunk 1 above)
Comment (chungthuang) [reply api/idea-card/models/idea-card.settings.json:68]: Is this to store response for the `Anything else you would like to share with us?` question?

Comment (alicefate) [reply api/idea-card/models/idea-card.settings.json:68]: yes. extraInfo is for that quesition.

File: api/idea-card/models/idea-card.settings.json [hunk 3]
```diff
@@ -19,23 +19,15 @@
...
     },
     "description": {
       "type": "text",
       "required": true
     },
     "targetAudience": {
       "type": "text",
-      "required": true
-    },
-    "hourCommitmentMin": {
```
Comment (chungthuang) [inline api/idea-card/models/idea-card.settings.json:32]: @dbradham can you double check this is not used anymore?

**Review by chungthuang — APPROVED** [review]

**Comment by chungthuang** [comment]
The change looks good, but I'll wait for @dbradham to review the schema.

File: api/idea-card/models/idea-card.settings.json (hunk 3 above)
Comment (dbradham) [reply api/idea-card/models/idea-card.settings.json:32]: I can confirm this.

**Review by dbradham — APPROVED** [review]

**Comment by dbradham** [comment]
@alicefate let me know if you need any help to solve these conflicts.

**Pull request merged**